tests/doc/property/double.cpp

```diff
@@ -11,15 +11,15 @@ TEST_CASE() {
     // [example]
     auto property = Property(45.2);
 
-    assert(property.get_kind() == Property::DOUBLE);
+    assert(property.kind() == Property::DOUBLE);
     assert(property.as_double() == 45.2);
 
     // Various overload convert integers to double
-    assert(Property(45).get_kind() == Property::DOUBLE);     // int
-    assert(Property(45l).get_kind() == Property::DOUBLE);    // long
-    assert(Property(45ll).get_kind() == Property::DOUBLE);   // long long
-    assert(Property(45u).get_kind() == Property::DOUBLE);    // unsigned
-    assert(Property(45ul).get_kind() == Property::DOUBLE);   // unsigned long
-    assert(Property(45ull).get_kind() == Property::DOUBLE);  // unsigned long long
+    assert(Property(45).kind() == Property::DOUBLE);     // int
```

Comment:
Should integer properties be added?

Reply:
Most integers can be represented as a double value, so I don't think it is worth it.

Reply:
Fair enough.